molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/entity/impl/GavinAnnotator.java

```diff
@@ -0,0 +1,351 @@
+package org.molgenis.data.annotation.entity.impl;
+
+import autovalue.shaded.com.google.common.common.collect.Iterables;
+import org.apache.commons.lang.StringUtils;
+import org.molgenis.MolgenisFieldTypes;
+import org.molgenis.data.AttributeMetaData;
+import org.molgenis.data.DataService;
+import org.molgenis.data.Entity;
+import org.molgenis.data.MolgenisDataException;
+import org.molgenis.data.annotation.cmd.EffectsAnnotator;
+import org.molgenis.data.annotation.entity.AnnotatorInfo;
+import org.molgenis.data.annotation.entity.EntityAnnotator;
+import org.molgenis.data.annotation.entity.impl.SnpEffAnnotator.Impact;
```

Comment:
I will put this outside SnpEffAnnotator. Because it is used by multiple annotators.

public enum Impact
	{
		MODIFIER, LOW, MODERATE, HIGH
	}